### PR TITLE
Feature/only visible for user post in channel

### PR DESCRIPTION
**Pull request by jcalero23**
* Set message visibility to ephemeral so that messages are only visible for the user
* Added a **Post in channel** button to post the message in the same channel

### CODE REVIEW

File: lambda/add_meaning.py [hunk 1]
```diff
@@ -475,24 +498,32 @@ def lambda_handler(event, context):
 
     # Check which action was sent
     if event_type == "block_actions" and actions is not None:
-        # Obtain required data
+        # Obtain the channel id i.e approver id
```
Comment (gihernandez) [inline lambda/add_meaning.py:501]: You can update the comment (remove the "i.e approver id" since this does not apply for all cases anymore)

**Review by gihernandez — CHANGES_REQUESTED** [review]

File: lambda/add_meaning.py [hunk 2]
```diff
@@ -475,24 +498,32 @@ def lambda_handler(event, context):
...
+        channel = payload['channel']['id']
+        
+        action_id = actions[0]['action_id']
+        
+        if action_id == 'Post':
```
Comment (oijusti) [inline lambda/add_meaning.py:506]: Can _`action_id`_ be a little bit more descriptive like _`post_in_channel`_ or something?

File: lambda/explain.py [hunk 3]
```diff
@@ -75,6 +75,24 @@ def explain(acronym):
...
+                    "elements": [
+                        {
+                            "type": "button",
+                            "text": {
+                                "type": "plain_text",
+                                "text": "Post in channel"
+                            },
+                            "style": "primary",
+                            "value": item['Acronym'],
+                            "action_id": "Post"
```
Comment (oijusti) [inline lambda/explain.py:93]: and here too

**Review by oijusti — APPROVED** [review]

File: lambda/add_meaning.py (hunk 1 above)
Comment (jcalero23) [reply lambda/add_meaning.py]: Done! Thanks

File: lambda/add_meaning.py (hunk 2 above)
Comment (jcalero23) [reply lambda/add_meaning.py]: Done! Thanks

File: lambda/explain.py (hunk 3 above)
Comment (jcalero23) [reply lambda/explain.py]: Done! Thanks

**Review by gihernandez — APPROVED** [review]

**Pull request merged**